tests/python/contrib/test_ethosn/test_partition_params.py

```diff
@@ -0,0 +1,104 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Ethos-N partition parameter tests"""
```

Comment:
It would be good to have tests also exercising parameters such as `tops` and `ple_ratio` when asking for an `ethos-n78` target, so that we can see which errors to expect from the underlying layers of the codegen.

Reply:
I've added no code checking tops or ple_ratio, so those values are not checked here.